sqlite/src/vdbe.c

```diff
@@ -1935,6 +1935,20 @@ case OP_Remainder: {           /* same as TK_REM, in1, in2, out3 */
     pOut->z = pOut->zMalloc = NULL;
   }
 
+  /* Subtraction only: If one operand is a String and the other is a Datetime,
+     attempt to promote the String operand to Datetime. */
+  if( ( pOp->opcode==OP_Subtract )&&( (pIn1->flags|pIn2->flags)&MEM_TypeMask )==(MEM_Str|MEM_Datetime) ) {
```

Comment:
Enclose this in a ```#ifdef SQLITE_BUILDING_FOR_COMDB2``` block.

Reply:
It's already protected by `SQLITE_BUILDING_FOR_COMDB2`.

Reply:
Yes, I see it now. Wasn't visible in the patch.